### DISCUSSION TITLE
Some checks were not successful

### DISCUSSION THREAD

**Discussion by dhruvparmar2310** [community/community#32578] [Other Feature Feedback, Questions, & Ideas] [answered]
Why I am not getting this image.
Please help me in this, need urgent help.
Please give me the solution of this, if any one knows with step by step to solve it.
Due to this I am not able to get favicon icon in phone device, on google search of my website....

![Screenshot 2022-09-10 223543](https://user-images.githubusercontent.com/65495463/189494334-79a8a483-d288-43d5-a342-bcddb7d955b3.png)

**Comment by airtower-luna** [comment, marked as answer]
Check the job logs by clicking on "details", the workflow summary there tells you what's going on. It seems like a bunch of builds were canceled because you made a new commit before the previous one was done and the build & deploy process for the new commit canceled the old one.

**Reply by dhruvparmar2310** [reply]
Thanks a lot, I understood; and now I need to check this at every time on push.
